.github/workflows/generate-and-publish-sdk-sources.yaml

```diff
@@ -18,7 +18,7 @@ on:
 
 jobs:
   generate-and-publish-sources:
-    uses: ExpediaGroup/expediagroup-java-sdk/.github/workflows/selfserve-full-workflow.yaml@main
+    uses: ExpediaGroup/expediagroup-java-sdk/.github/workflows/selfserve-full-workflow.yaml@v20241009
```

Comment:
why not `main`?

Reply:
So that we don't break the generation if the main branch introduces breaking changes